random.js

```diff
@@ -0,0 +1,76 @@
+var slideIndex = 0;
+var slides = document.getElementsByClassName("mySlides");
```

Comment:
"mySlides" is camelCase and "slide-inactive" is a different notation. I would choose 1 and stick with it